docker/ml/verify.py

```diff
@@ -1,5 +1,4 @@
-import tensorflow
-import keras
```

Comment:
You install Keras in the docker so you have to make sure it import it correctly. 
Btw the model I am building for system diagnostic uses Keras (I don't like Keras but since I believed your model use Keras I used it to build it). Not sure if this model is going somewhere but let's make sure this docker can use keras and pytorch. It will be hard to add it afterward